__tests__/app.test.js

```diff
@@ -321,3 +319,49 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe("GET /api/articles (sorting queries)", () => {
+  test("200 - responds with all articles sorted by any column in order given", () => {
+    return supertest(app)
+      .get("/api/articles?sort_by=title&order=desc")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles.length).toBeGreaterThan(0);
+        expect(articles).toBeSortedBy("title", { descending: true });
+      });
+  });
+  test("200 - responds with all articles sorted by created_at (default) given the order only", () => {
+    return supertest(app)
+      .get("/api/articles?order=asc")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles.length).toBeGreaterThan(0);
+        expect(articles).toBeSortedBy("created_at", { ascending: true });
```

Comment:
According to the [jest-sorted](https://www.npmjs.com/package/jest-sorted) docs, there is no such property as {ascending: true} , we instead get this effect by setting descending to false (which it actually defaults to!)